deployments/properties/azure-kubernetes-service-kompose.properties.json

```diff
@@ -0,0 +1,7 @@
+{
+    "name": "Build and Deploy an app to AKS with Kompose",
```

Comment:
This name looks big, can we have smaller name for eg: Deploy to AKS with Kompose

Reply:
Shortened